src/sentry/templates/sentry/emails/reports/body.html

```diff
@@ -363,7 +363,7 @@ <h1>
 
     {% with series=report.projects.series height=110 %}
 
-    <h4>Errors by Project</h4>
+    <h4>Events by Project</h4>
```

Comment:
The bar chart including transactions now just magically works, since it's just a sum of the `report.series` values.

Reply:
Does it though
```
    error_series = [
        # Drop the transaction count from each series entry
        (to_datetime(timestamp), Point(*values[:2]))
        for timestamp, values in report.series
    ]
```
I thought we dropped transaction count from the series entry?